src/OpenApi/test/Microsoft.AspNetCore.OpenApi.Tests/Integration/snapshots/OpenApi3_0/OpenApiDocumentIntegrationTests.VerifyOpenApiDocument_documentName=controllers.verified.txt

```diff
@@ -118,10 +118,12 @@
         "type": "object",
         "properties": {
           "title": {
-            "type": "string"
+            "type": "string",
```

Comment:
STJ's schema generation assumes strings are nullable in an oblivious nullability context (see <Nullable>disable</Nullabe> in Sample.csproj), hence the delta here.